### PR TITLE
chore(starters): gatsby-london-night-and-day

### PR DESCRIPTION
**Pull request by jooplaan**
## Description

Adding a new Starter theme

### Documentation

https://github.com/jooplaan/gatsby-london-night-and-day

### CODE REVIEW

**Comment by laurieontech** [comment]
Thanks for the PR! The theme is beautiful. 

I noticed the README included npx installation instructions. Is there a reason for that? We try to stay consistent and use npm.

**Comment by jooplaan** [comment]
Removed mentioning npx in the readme. Not needed.

**Comment by laurieontech** [comment]
Looks like a test is failing because of one of the tags. "Styling: SASS" is not valid. If you remove that, or change it to Styling: Other. It should be good to go!

File: docs/starters.yml [hunk 1]
```diff
@@ -5066,6 +5066,26 @@
...
+- url: https://london-night-day.netlify.com/
+  repo: https://github.com/jooplaan/gatsby-london-night-and-day
+  description: A custom, image-centric dark and light mode aware theme for Gatsby. Advanced from the Gatsby starter London After Midnight.
+  tags:
+    - Blog
+    - Portfolio
+    - Gallery
+    - SEO
+    - Markdown
+    - Sass/SCSS
```
Comment (laurieontech) [inline docs/starters.yml]:
```suggestion
    - Styling:SCSS    
```
In order for tests to pass it needs to match the specific tag.

**Comment by laurieontech** [comment]
Thanks for staying with me on this one. Let's get this merged!

**Pull request merged**